WordPress/gutenberg
# Use WP_TESTS_EMAIL and WP_TESTS_TITLE when creating environments using wp-env

Fixes #42419

## What?
This PR updates the `wp-env` package to use the `WP_TESTS_EMAIL` and `WP_TESTS_TITLE` when creating the development and test environments. If those values are set to null in the config, then the current defaults are used instead.

It also updates the email in the readme to match the current reality.

## Why?
The docs indicate that this should be possible, but this isn't the case.

## How?
`WP_TESTS_EMAIL` and `WP_TESTS_TITLE` are read and passed to the install command to be used to create the environments.

## Testing Instructions
1. Clone this repo
2. Navigate to the `env` package using `cd packages/env`
3. Install dependencies using `npm i`
4. Create a `.wp-env.json` file in the root of the package folder with the contents below:
```
{
	"phpVersion": "7.4",
	"core": null,
	"plugins": [],
	"config": {
		"WP_TESTS_EMAIL": "admin@wpenv.com",
		"WP_TESTS_TITLE": "Development"
	},
	"env": {
		"development": {},
		"tests": {
			"config": {
				"WP_TESTS_EMAIL": "test@wpenv.com",
				"WP_TESTS_TITLE": "Testing"
			}
		}
	}
}

```
6. Build your environment by running `./bin/wp-env`.
7. Navigate to `localhost:8888` / `localhost:8889` and observe that the blog title matches what is defined in `.wp-env.json` for the respective environment.
8. Navigate to `localhost:8888/wp-admin/users.php` / `localhost:8889/wp-admin/users.php` and observe that the admin email matches what was defined in `.wp-env.json` for the respective environment`
